### PR TITLE
node.js: leading slash in the path argument of rest-style api

### PR DESCRIPTION
**Pull request by etimr**

### CODE REVIEW

File: node.js/core-services.md [hunk 1]
```diff
@@ -1125,7 +1125,7 @@ srv.patch('/Books',...)   -->  srv.send('PATCH','/Books',...)
 srv.delete('/Books',...)  -->  srv.send('DELETE','/Books',...)
 ```
 
-You can also use them as REST-style variants to run queries by omitting the leading slash in the `path` argument, or by passing a reflected entity definition instead. In that case they start constructing *bound* [`cds.ql` query objects](cds-ql), as their [CRUD-style counterparts](#crud-style-api):
+Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. But you can also use these methods as REST-style variants to run queries by omitting the leading slash in the `path` argument, or by passing a reflected entity definition instead. In that case they start constructing *bound* [`cds.ql` query objects](cds-ql), as their [CRUD-style counterparts](#crud-style-api):
```
Comment (johannes-vogel) [inline node.js/core-services.md:1128]: I think this paragraph is overly complex. Maybe we can simplify but I don't have a good idea atm....

Comment (etimr) [reply node.js/core-services.md]: @johannes-vogel could you please check the simplified variant?

File: node.js/core-services.md [hunk 2]
```diff
@@ -1125,7 +1125,7 @@ srv.patch('/Books',...)   -->  srv.send('PATCH','/Books',...)
 srv.delete('/Books',...)  -->  srv.send('DELETE','/Books',...)
 ```
 
-You can also use them as REST-style variants to run queries by omitting the leading slash in the `path` argument, or by passing a reflected entity definition instead. In that case they start constructing *bound* [`cds.ql` query objects](cds-ql), as their [CRUD-style counterparts](#crud-style-api):
+Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. Omitting the leading slash or passing a reflected entity definition instead, leads to construction of *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):
```
Comment (johannes-vogel) [inline node.js/core-services.md:1128]:
```suggestion
Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. Omitting the leading slash or passing a reflected entity definition instead, constructs *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):
```

Comment (johannes-vogel) [reply node.js/core-services.md:1128]: @renejeglinsky could you please also double check the wording?

File: node.js/core-services.md [hunk 3]
```diff
@@ -784,7 +784,7 @@ return : result of this.dispatch(req)
 Use this method to send synchronous requests to a service for execution.
 
 -  `method` can be a HTTP method, or a name of a custom action or function
```
Comment (renejeglinsky) [inline node.js/core-services.md:786]:
```suggestion
-  `method` can be an HTTP method, or a name of a custom action or function
```

File: node.js/core-services.md [hunk 4]
```diff
@@ -784,7 +784,7 @@ return : result of this.dispatch(req)
...
--  `path` can be an arbitrary URL, starting with a leading `'/'`
+-  `path` can be an arbitrary URL, starting with a leading `'/'`, it is passed to a service without any modification as a string.
```
Comment (renejeglinsky) [inline node.js/core-services.md:787]:
```suggestion
-  `path` can be an arbitrary URL, starting with a leading `'/'`, it is passed to a service without any modification as a string
```

File: node.js/core-services.md [hunk 5]
```diff
@@ -1125,7 +1125,7 @@ srv.patch('/Books',...)   -->  srv.send('PATCH','/Books',...)
 srv.delete('/Books',...)  -->  srv.send('DELETE','/Books',...)
 ```
 
-You can also use them as REST-style variants to run queries by omitting the leading slash in the `path` argument, or by passing a reflected entity definition instead. In that case they start constructing *bound* [`cds.ql` query objects](cds-ql), as their [CRUD-style counterparts](#crud-style-api):
+Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent to a service unmodified. Omitting the leading slash or passing a reflected entity definition instead, constructs *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):
```
Comment (renejeglinsky) [inline node.js/core-services.md:1128]:
```suggestion
Leading slash in the `path` argument results in the same behaviour as in `srv.send()`: `path` is sent unmodified to a service. Omitting the leading slash, or passing a reflected entity definition instead, constructs *bound* [`cds.ql` query objects](cds-ql), equivalent to [CRUD-style API](#crud-style-api):
```

**Review by renejeglinsky — APPROVED** [review]

**Pull request merged**